### PR TITLE
Migrate PRBT ikfast plugin from moveit_resources

### PR DESCRIPTION
**Pull request by v4hn**
As announced in https://github.com/ros-planning/moveit/issues/2653#issuecomment-850399443 .
Together with https://github.com/ros-planning/moveit_resources/pull/79 this is supposed to provide a straight-forward solution for #2653 .

Moving the plugin here circumnavigates problems with running tests in the moveit repository that are based on it. Before, this plugin was part of moveit_resources, but introduced a compile time dependency on moveit_core.
Because a repository's tests should run without having to compile downstream code (a sane requirement on buildfarm/prerelease tests), we move the plugin here instead.

### CODE REVIEW

File: moveit_kinematics/prbt_manipulator_ikfast_plugin/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package moveit_resources_prbt_ikfast_manipulator_plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.8.0 (2021-04-28)
+------------------
+
+0.7.2 (2021-03-26)
```
Comment (rhaschke) [inline moveit_kinematics/prbt_manipulator_ikfast_plugin/CHANGELOG.rst:8]: The version info here doesn't match to versions of MoveIt anymore.

**Review by rhaschke — CHANGES_REQUESTED** [review]
As said before, I do not support moving a _specific_ IKfast planner into the `moveit_kinematics` package, which should provide `generic` interfaces only.
Additionally, as pointed out in https://github.com/ros-planning/moveit_resources/pull/79#discussion_r642044458, moving the IK plugin into `moveit_kinematics`, will not resolve the issue that `moveit_resources` depends on `moveit_kinematics`.

**Comment by v4hn** [comment]
> As said before, I do not support moving a specific IKfast planner into the moveit_kinematics package, which should provide generic interfaces only.

We could put it in a separate ROS package in the repository if that's your main concern.
The only important thing to get this to work is that the code is *somewhere* in this repository and it somewhat made sense to me to associate it with `moveit_kinematics`.

**Comment by v4hn** [comment]
Closing as we decided to take [a different approach](https://github.com/ros-planning/moveit/issues/2653#issuecomment-856000201).
Still, someone has to actually do all the work for this...

**Pull request closed without merging**